cypress/integration/view-and-manage-evidence.spec.ts

```diff
@@ -201,7 +201,9 @@ describe('When a user inputs a validity date that is in the past', () => {
       cy.get('button').contains('Yes, accept').click();
 
       //assert
-      cy.get('span').eq(0).should('contain', 'The date cannot be in the past.');
+      cy.get('fieldset>span')
```

Comment:
This plus a few other tests are pretty flakey. I think this seems to have fixed it.